File: javascript/json_numbers.mjs
// Script takes args and outputs a list of lengths

const myStrings = process.argv.slice(2)

// Create an array of numbers based on the length of the string args
const stringLengths = myStrings.map((string) => string.length)

const jsonString = JSON.stringify(stringLengths)

console.log(jsonString)

File: swift/json_numbers.py
"""Script takes args and outputs a list of lengths"""
import json
import sys


my_strings = sys.argv[1:]

# Create an array of numbers based on the length of the string args
string_lengths = [len(string) for string in my_strings]

# Cast to JSON and print to stdout
print(json.dumps(string_lengths))
